.github/workflows/run-external-processing.yml

```diff
@@ -67,9 +67,12 @@ jobs:
       - name: Run EXTERNAL_PROCESSING scenario
         run: ./run.sh EXTERNAL_PROCESSING
 
+      - name: Run EXTERNAL_PROCESSING_BLOCKING scenario
+        run: ./run.sh EXTERNAL_PROCESSING_BLOCKING
+
       - name: Compress logs
         id: compress_logs
-        if: always() && steps.build.outcome == 'success'
+        if: always()
```

Comment:
Could you keep that and add `id` on the build agent step ? The idea is to run this step if any `run.sh` step is executed.

Reply:
Applied in f96a2654c0e1d380e215c7cc582457458d1de5e9 😄